make.sh

```diff
@@ -86,12 +86,27 @@ if [ -z "$project_name" ] ; then
     read -p "PROJECT NAME (i.e. NewProject):" project_name
 fi
 
+if [[ -z "$minimum_ios_version" ]]; then
+    read -p "iOS Minimum Version (i.e. 14.0):" minimum_ios_version
+fi
+
+# Enforce minimum version
+version_regex='^[0-9_]+(\.[0-9]+)+$'
+if ! [[ $minimum_ios_version =~ $version_regex ]]; then
+    echo "=> Minimum version incorrect. Reverting to default version."
+    minimum_ios_version="14.0"
+fi
```

Comment:
@phongvhd93 Do we still need `--ios-version 14.0` when calling `make.sh`? If not, please update the call in `test.yml` and the README content 🙏 

```
./make.sh --bundle-id co.nimble.kmm.sample --bundle-id-staging co.nimble.kmm.sample.staging --project-name sample --ios-version 14.0
```

Reply:
We need to the parameter `--ios-version 14.0` when calling `make.sh`. in case the developer mistyping, the script will continue to generate with default iOS version